### DISCUSSION TITLE
Remote Config: addOnConfigUpdateListener()

### DISCUSSION THREAD

**Discussion by hkaras19** [invertase/react-native-firebase#7075] [Ideas]
I am looking to add remote config to my react native app, and I would like for the remote config values to be accessed instantly by users. This technically can be accomplished by setting the cache length to 0; however, I don't want to run into rejected requests from Firebase as this isn't the most efficient solution. While looking through the documentation (linked below), there appears to be a function provided by firebase to listen for remote config changes and instantly update the local values. I was wondering if there are plans to add this to react-native-firebase as I don't believe it is a current feature. This would be very useful for managing the release of certain features.

https://firebase.google.com/docs/remote-config/get-started?platform=ios#add-real-time-listener

**Comment by cgorrieri** [comment]
It seems to be a recent feature, but I would love to see it here. I also believe this will ease some implementations!
